.idea/misc.xml

```diff
@@ -1,26 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="EntryPointsManager">
```

Comment:
You need not commit changes to this file. Can you please remove it from the commit?